PcsConfig/DeviceGroupsTests.cs

```diff
@@ -0,0 +1,280 @@
+﻿using Helpers.Http;
```

Comment:
nit: Remove Pcs from the file names and folder names

Reply:
done